### DISCUSSION TITLE
Can't obtain certificate

### DISCUSSION THREAD

**Discussion by Terrortorpe** [nextcloud/all-in-one#1700] [Questions] [answered]
Hi,

I'm using the docker image for the AIO followed the instructions, I am not using a reverse proxy and I have opened ports 80 and 8443 as well as 443 on my router and pointed them to the correct device.

I have configured my domain to point to the correct IP address which I confirmed with dig and I get the following error on the AIO configuration page:

The server is not reachable on Port 443. You can verify this e.g. with 'https://portchecker.co/' by entering your domain there as ip-address and port 443 as port.

I have checked that port 443 is open the way the message suggests. Portchecker says it is open. I do not want to use the domain verification skip as I feel like I am missing something.

I don't believe this to be a bug I think I might be missing something so I'm posting it as a question

Help would be appreciated

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation
